portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,154 @@
     <script src="script.js"></script>
   </head>
   <body>
+    <div id = "banner">
+        <h1>Hello World!</h1>
+        <div id="banner-content">Cirque de Towers, Wyoming</div>
+    </div>    
+
+    <div id = "profile-pic">
+        <img style = "width: 200px; height = 200px" src="/images/siteimages/climbing.jpg">
+    </div>
+
+    <div id = "side-pic">
+        <img src = "/images/siteimages/truck.jpg" style = "width: 250px; height = 250px;">
+    </div>
+
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <p style = "padding: 20px">
+          My name is Meleah, but I go by Mel. I love hiking, climbing, and working 
+          on projects. I am studying Computer Engineering, but find myself just as
+          excited about learning to code! I love being outdoors and exploring the 
+          mountains. Maybe it's a side effect of growing up in the flat state of 
+          Texas, but going to the mountains always brings me so much joy. I plan to
+          attempt the Seven Summits one day. I also enjoy working on old cars and have
+          been fixing up my truck in my free time.
+      </p>
+    </div>
+
+<!-- I originally had this as div row/cols but I was struggling to get the 
+    level of customication I was looking for, so I redid it as a table -->
+    <table>
+        <tr>
+            <td></td>
+            <td></td>
+            <td id = "rightcol" style = "font-size: 20px;"> Adventure Map!</td>
```

Comment:
I'll use class instead of id for them. class is a group of elements that as the same styling, and id is for a specific item.

Reply:
Improved code by switching table ids to classes